generators/generator-bot-adaptive/generators/app/index.js

```diff
@@ -249,7 +249,7 @@ module.exports = class extends BaseGenerator {
   _writeJsPackageJson() {
     const { botName, integration } = this.options;
 
-    const sdkVersion = '~4.13.2-preview';
+    const sdkVersion = '~4.13.3-preview';
```

Comment:
We may as well remove the tilde for now since it isn't buying us anything.

Reply:
Fixed.